src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -502,7 +504,7 @@ private static BidderRequest createBidderRequest(String bidder, BidRequest bidRe
         return BidderRequest.of(bidder, bidRequest.toBuilder()
                 .user(privacyEnforcementResult.getUser())
                 .device(privacyEnforcementResult.getDevice())
-                .regs(privacyEnforcementResult.getRegs())
+                .regs(regs)
```

Comment:
You can simply skip extracting `Regs` from `bidRequest` to then simply put it back into it, especially since you are calling `bidRequest.toBuilder()` - it creates a new `BidRequest.Builder` object with all fields copied from `BidRequest` object and gives you an ability to replace them - so no point in replacing what is to remain the same)